CHANGES.md

```diff
@@ -42,6 +42,8 @@
 - Allow `][` in code blocks (@Julow, #1149)
   This was interpreted as "code blocks with result", which now mandate a delimiter:
   `{delim@lang[ code ]delim[ result ]}`
+- Output file paths in the man and latex backends changed to avoid name clashes
```

Comment:
```suggestion
- Output file paths and labels in the man and latex backends changed to avoid name clashes
```
Just so that if someone is using the latex backend and handwriting labels, they can find the cause of their new problem in the changelog.